packages/n4s/src/rules/isNegative/index.js

```diff
@@ -0,0 +1,10 @@
+import isNumeric from '../isNumeric';
+
+function isNegative(value, arg1) {
```

Comment:
You're not using `arg1`.
```suggestion
function isNegative(value) {
```

Reply:
Done.